esmvaltool/diag_scripts/emergent_constraints/ecs_cmip.cdl

```diff
@@ -0,0 +1,50 @@
+netcdf ECS {
```

Comment:
Please use yaml configuration files instead of cdl/netcdf, maybe this can simply go in the recipe? It seems to depend on the models you are using anyway, so would need changing if the model set changes.

Reply:
This is not something that should to into a recipe as these numbers are either the result of another diagnostic or taken from literature (user choice). NetCDF is easy to process with NCL whereas yaml is not. The cdl script is provided so that an advanced user can add data easily if needed.

Reply:
While this may be the easy solution for you as a programmer, I think it is not easy for users.